front/src/applications/stdcm/components/StdcmForm/StdcmConfig.tsx

```diff
@@ -173,6 +185,27 @@ const StdcmConfig = ({
     }
   }, []);
 
+  useEffect(() => {
+    if (isPathFindingLoading) {
+      setShowMessage(true);
+    }
+
+    const handleAnimationEnd = () => {
+      setShowMessage(false);
+    };
+
+    const element = document.querySelector('.pathFinding-status');
```

Comment:
It's better to use a `ref` when possible in React. You can declare it next to the other `useRef` of the component.
```suggestion
    const pathfindingBannerRef = useRef<HTMLDivElement>(null);
```

Reply:
Good catch ! 
Done !